spec/fixtures/linuxdev/bolt-kerberos-test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-mkdir -p ~/.puppetlabs/bolt
-rm -f ~/.puppetlabs/bolt/bolt.yaml
+mkdir -p ~/.puppetlabs/etc/bolt
+rm -f ~/.puppetlabs/etc/bolt/bolt-project.yaml
```

Comment:
This isn't a file loaded by Bolt. Is that intentional?

Reply:
Oh ha! Should be bolt-defaults I think. Thanks!